packages/base/src/dialogs/symbology/components/color_ramp/CanvasSelectComponent.tsx

```diff
@@ -72,16 +91,19 @@ const CanvasSelectComponent: React.FC<ICanvasSelectComponentProps> = ({
     const colorMapList: IColorMap[] = [];
 
     colorRampNames.forEach(name => {
+      const cmoRamp = (cmocean as any)[name];
+      const requiredShades =
+        Array.isArray(cmoRamp) && cmoRamp.length > 0 ? cmoRamp.length : 256;
```

Comment:
Do we need this custom code to handle cmocean colormaps now that we're using `colormap` as our interface to all colormaps?

Reply:
That seems right from my research too. I'm continuing to think about this -- my goal is to avoid conditional logic based on a colormap's presence in `cmocean.json`. :thinking: Maybe we reduce the number of samples in `cmocean.json` to 8.